### ISSUE TITLE
Error: Invalid Value used as weak map key

### ISSUE THREAD

**Issue by ElcomJ** [neoclide/coc.nvim#4438]
## Result from CocInfo

```
## versions

vim version: NVIM v0.9.0-dev-505+ga6b05cb75
node version: v16.17.0
coc.nvim version: 0.0.82-bd6bc442 2022-12-13 23:15:37 +0800
coc.nvim directory: C:\Users\elcomj\AppData\Local\nvim-data\plugged\coc.nvim
term: vtpcon
platform: win32

## Log of coc.nvim

2022-12-13T18:02:15.877 INFO (pid:11104) [extension:coc-tabnine] - Using tabnine from C:\Users\elcomj\AppData\Local\coc\extensions\coc-tabnine-data\binaries\4.4.200\x86_64-pc-windows-gnu\TabNine.exe
2022-12-13T18:02:15.946 INFO (pid:11104) [plugin] - coc.nvim initialized with node: v16.17.0 after 390
2022-12-13T18:02:16.068 WARN (pid:11104) [workspace] - workspace.createStatusBarItem is deprecated, please use window.createStatusBarItem instead. 
    at Object.t.initStatusBar (C:\Users\elcomj\AppData\Local\coc\extensions\node_modules\coc-spell-checker\out\index.js:1:155936)
    at Object.t.activate (C:\Users\elcomj\AppData\Local\coc\extensions\node_modules\coc-spell-checker\out\index.js:1:23162)
2022-12-13T18:02:16.079 INFO (pid:11104) [language-client-index] - Language server "cSpell" started with 5468
2022-12-13T18:02:16.588 INFO (pid:11104) [attach] - receive notification: runCommand [ 'explorer' ]
2022-12-13T18:02:16.622 INFO (pid:11104) [attach] - receive notification: doAutocmd [ 1 ]
2022-12-13T18:02:16.632 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_coc_helper_1',
  'BufDelete',
  2
]
2022-12-13T18:02:16.632 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_explorer_2',
  'BufDelete',
  2
]
2022-12-13T18:02:18.634 INFO (pid:11104) [attach] - Request action: doKeymap [ 'explorer-key-n-[cr]' ]
2022-12-13T18:02:18.648 INFO (pid:11104) [attach] - receive notification: doAutocmd [ 1 ]
2022-12-13T18:02:18.661 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_coc_helper_1',
  'BufDelete',
  1
]
2022-12-13T18:02:18.663 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_explorer_2',
  'BufDelete',
  1
]
2022-12-13T18:02:18.663 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_coc_helper_1',
  'BufWipeout',
  1
]
2022-12-13T18:02:18.664 INFO (pid:11104) [attach] - receive notification: runCommand [
  'coc-helper.internal.didVimEvent_m1_v0_15_2_explorer_2',
  'BufWipeout',
  1
]
2022-12-13T18:02:18.940 INFO (pid:11104) [attach] - receive notification: doAutocmd [ 1 ]
2022-12-13T18:02:19.497 INFO (pid:11104) [services] - LanguageClient Pyright Server state change: stopped => starting
2022-12-13T18:02:19.518 INFO (pid:11104) [language-client-index] - Language server "pyright" started with 10084
2022-12-13T18:02:19.578 INFO (pid:11104) [attach] - receive notification: runCommand [ 'explorer' ]
2022-12-13T18:02:20.104 INFO (pid:11104) [services] - LanguageClient Pyright Server state change: starting => running
2022-12-13T18:02:20.109 INFO (pid:11104) [services] - service pyright started
2022-12-13T18:02:22.750 ERROR (pid:11104) [completion-util] - Unexpected completion item from [object Object]: {
  label: undefined,
  sortText: '02.9999.main',
  kind: 9,
  filterText: undefined
}
2022-12-13T18:02:22.752 ERROR (pid:11104) [server] - unhandledRejection  Promise {
  <rejected> TypeError: Invalid value used as weak map key
      at WeakMap.set (<anonymous>)
      at C:\Users\elcomj\AppData\Local\nvim-data\plugged\coc.nvim\build\index.js:257:38170
      at Array.map (<anonymous>)
      at C:\Users\elcomj\AppData\Local\nvim-data\plugged\coc.nvim\build\index.js:257:38120
} TypeError: Invalid value used as weak map key
    at WeakMap.set (<anonymous>)
    at C:\Users\elcomj\AppData\Local\nvim-data\plugged\coc.nvim\build\index.js:257:38170
    at Array.map (<anonymous>)
    at C:\Users\elcomj\AppData\Local\nvim-data\plugged\coc.nvim\build\index.js:257:38120
2022-12-13T18:02:27.349 WARN (pid:11104) [completion-complete] - Completion timeout after 5000ms [ 'python-import' ]
2022-12-13T18:02:33.778 INFO (pid:11104) [attach] - receive notification: showInfo []
```

## Describe the bug

Good afternoon, after running a `PlugUpdate` and updating the plugin to the latest version, probably the latest commit bd6bc44 came with a problem.

In a **Python** file, if I insert a `space` just after the Keywords `import` and `from`, the error `Invalid Value used as weak map key` appears (I will insert images below), and almost certainly the error came with the last commit, because I was using it before in the previous commit and it was not happening.

## Reproduce the bug

https://user-images.githubusercontent.com/37147897/207448013-9e2fb775-3be3-4ac9-a86e-307ba14a094a.mp4

## Screenshots (optional)

![1](https://user-images.githubusercontent.com/37147897/207448497-fb6c9e78-95e1-4d8a-aa23-9cddfa8ceda4.PNG)
![2](https://user-images.githubusercontent.com/37147897/207448504-b8e8bb3b-a5e1-4b61-bfa6-2e52ffc678ed.PNG)

**Comment by fannheyward** [comment]
Reproduced.

**Comment by fannheyward** [comment]
Fixed in coc-pyright v1.1.284, upgrade and try again.

**Comment by ElcomJ** [comment]
Good morning, thanks for the help, as mentioned the problem has been fixed in version `v1.1.284` of `coc-pyright`.

Thanks for the great work and have a nice day :)
